### PR TITLE
change focus color instead of opacity of video play button

### PR DESCRIPTION
**Pull request by Pandoraaa**
This PR will highlight the central play button on a video when it is focused in order to improve a11y. 

image

### CODE REVIEW

File: client/src/app/menu/language-chooser.component.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class LanguageChooserComponent {
     @Inject(LOCALE_ID) private localeId: string
   ) {
     const l = Object.keys(I18N_LOCALES)
-                    .map(k => ({ id: k, label: I18N_LOCALES[k] , iso: getShortLocale(k)}))
+                    .map(k => ({ id: k, label: I18N_LOCALES[k] , iso: getShortLocale(k) }))
```
Comment (rigelk) [inline client/src/app/menu/language-chooser.component.ts:23]: Thanks @Pandoraaa! That should fix the lint stage.

File: client/src/sass/player/peertube-skin.scss [hunk 2]
```diff
@@ -88,6 +88,10 @@ body {
     &:hover {
       opacity: 0.8;
     }
+
+    &.focus-visible{
+      background-color: var(--mainColor);
+    }
```
Comment (rigelk) [inline client/src/sass/player/peertube-skin.scss:94]: why not combine `:hover` and `.focus-visible` classes, keeping either the `opacity` or `background-color`?

Comment (Pandoraaa) [reply client/src/sass/player/peertube-skin.scss]: Oh thanks I didn't think of that !

**Comment by rigelk** [comment]
Thanks @Pandoraaa!

**Pull request merged**